src/CMakeLists.txt

```diff
@@ -10,7 +11,6 @@ target_include_directories(jsonbuilder
         $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
         $<INSTALL_INTERFACE:include>)
 
-target_link_libraries(jsonbuilder PUBLIC uuid::uuid)
```

Comment:
Note: `libuuid` isn't supported on Windows. But we don't really need it -- we only need to be able to accept it as input, and since `uuid_t` is just a typedef for `char unsigned[16]`, we can easily accept `char unsigned const*` input parameters without depending on `libuuid`.